src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,4 +1,25 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "USERS")
```

Comment:
tu chyba nie będzie users, ale ta encja jest do usunięcia wiec chyba nieistotne

Reply:
Racja, ech to kopiuj-wklej